docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+---
+title: MSBuild properties for Microsoft.NET.Sdk
+description: Reference for the MSBuild properties that are understood by the .NET Core SDK.
+ms.date: 02/02/2020
+ms.topic: reference
+---
+# MSBuild properties for .NET Core SDK projects
+
+This page describes MSBuild properties for configuring .NET Core projects.
+
+> [!NOTE]
+> This page is a work in progress and does not list all of the useful MSBuild properties for the .NET Core SDK.
+
+## Framework and dependency properties
+
+- [NetStandardImplicitPackageVersion](#netstandardimplicitpackageversion)
+- [PackageReference](#packagereference)
+- [PackageTargetFallback](#packagetargetfallback)
+- [RuntimeIdentifier](#runtimeidentifier)
+- [RuntimeIdentifiers](#runtimeidentifiers)
+- [TargetFramework](#targetframework)
+- [TargetFrameworks](#targetframeworks)
+
+### NetStandardImplicitPackageVersion
+
```

Comment:
This is old, and doesn't apply to projects using `netstandard2` and up. Should this be less prominent, and with a note about applicability to `netstandard1.x` only?